Azure/azure-cli
# [ARM] `az data-boundary` Add DataBoundary Cmdlets

**Related command**


**Description**

This PR enables databoundary commands to create and get databoundary.

**Testing Guide**

[ARM] `az data-boundary show`: Will get the data boundary at a subscription or rg level
[ARM] `az data-boundary show-tenant`: Will get the data boundary at a tenant level
[ARM] `az data-boundary create`: Will put a data boundary at a tenant level (test will create a databoundary on existing databoundary which has already been created and will check for the proper error message)

**History Notes**


[ARM] `az data-boundary show`: Add command to support getting the data boundary at a specified scope
[ARM] `az data-boundary show-tenant`: Add command to support getting the data boundary at a tenant level
[ARM] `az data-boundary create`: Add command to support creating tenant to data boundary

---

This checklist is used to make sure that common guidelines for a pull request are followed.
